.gitignore

```diff
@@ -24,3 +24,4 @@ genesis.ssz
 venv
 
 *.ssz
+.idea
```

Comment:
fine to add, but would recommend putting this in a global `.gitignore`

Reply:
TIL global gitignores exist